chain/types/ethtypes/eth_transactions.go

```diff
@@ -37,7 +37,8 @@ type EthTx struct {
 	Gas                  EthUint64   `json:"gas"`
 	MaxFeePerGas         EthBigInt   `json:"maxFeePerGas"`
 	MaxPriorityFeePerGas EthBigInt   `json:"maxPriorityFeePerGas"`
-	V                    EthBigInt   `json:"v"`
+	AccessList           []EthHash   `json:"accessList"`
+	V                    EthBigInt   `json:"yParity"`
```

Comment:
Afaict what we put here is in fact YParity according to https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2098.md; V is only used for legacy transactions

This value seems to be set to 0/1; not 27/28 which are V values

Not sure if it makes sense to rename the V field to `YParity` as well, no strong opinions.

Reply:
I'd prefer to rename it to yParity as well to keep them consistent.